src/Components/Admin/Rooms/roomModal.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+const useStyles = makeStyles(() => ({
+    textField: {
+        width: "500px",
+    },
+    buttonContainer: {
+        display: "flex",
+        justifyContent: "space-around",
+    },
+}));
+
+export default function RoomModal({
+    modalOpen,
+    handleModalClose,
+    handleSubmit,
+}) {
+    const classes = useStyles();
+    const [roomName, setRoomName] = React.useState("");
+    const [roomDescription, setRoomDescription] = React.useState("");
+
+    const handleRoomNameChange = (event) => {
+        setRoomName(event.target.value);
+    };
+
+    const handleRoomDescriptionChange = (event) => {
+        setRoomDescription(event.target.value);
+    };
+
+    const handleModalCloseClick = () => {
+        setRoomName("");
+        setRoomDescription("");
```

Comment:
Why are we calling it here as well? Doesn't make sense in either function imo

Reply:
Same comment as above